### PR TITLE
Added fully e2e endpoint tests

### PR DESCRIPTION
**Pull request by mauriciofreshw**
POC for HCAP-369 and GH Actions test.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -32,33 +32,15 @@ jobs:
...
+
 
-      - name: NPM Test
-        working-directory: ./server
-        env:
-          POSTGRES_DB: db_test
-          POSTGRES_USER: admin
-          POSTGRES_PASSWORD: admin
-        run: |
-          npm ci
```
Comment (fwkendall) [inline .github/workflows/test.yml:63]: we should make this this is added to the makefile local-server-tests if we're removing it from here 😄

File: .docker/keycloak/users.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
+[
```
Comment (fwkendall) [inline .docker/keycloak/users.json:1]: curious how this was initially created - to add more users would I just mock them out here and trust the import/export scripts to manage things from there, seeing as the IDs aren't used during import?

File: .github/workflows/test.yml (hunk 1 above)
Comment (mauriciofreshw) [reply .github/workflows/test.yml:63]: _Dockerfile.dev_ is already doing this work for us

File: .docker/keycloak/users.json (hunk 2 above)
Comment (mauriciofreshw) [reply .docker/keycloak/users.json:1]: You can do it either this way or via UI, just make sure that all the ids under the `roles` field are correct. And if you choose to change via UI don’t forget to `make kc-export-users` afterward.

Comment (fwkendall) [reply .docker/keycloak/users.json:1]: Gotcha, so first time you add new users would mean following the README steps to get into the local KC and set them up for export 😄  makes sense!

**Review by fwkendall — APPROVED** [review]

**Review by ashtonmeuser — APPROVED** [review]

**Pull request merged**